### PR TITLE
[RADOS] Adding missing suite file and entries in pipeline metadata for 5.3, 6.0 and 6.1

### PR DESCRIPTION
**Pull request by neha-gangadhar**
[RADOS] Adding missing suite file and entries in pipeline metadata for 5.3, 6.0 and 6.1

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: pipeline/metadata/5.3.yaml [hunk 1]
```diff
@@ -876,6 +876,36 @@ pipelines:
...
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - rados
+        - name: "Tier-4 RADOS test pool osd recovery"
+          execution_time: ""
+          suite: "suites/pacific/rados/tier-4_rados_test-pool-osd-recovery.yaml"
+          global-conf: "conf/pacific/rados/11-node-cluster.yaml"
```
Comment (pdhiran) [inline pipeline/metadata/5.3.yaml:902]: The suite needs to be changed to : ` 13-node-cluster.yaml ` , as this requires node replacements.

File: pipeline/metadata/6.0.yaml [hunk 2]
```diff
@@ -791,6 +791,36 @@ pipelines:
...
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - rados
+        - name: "Tier-4 RADOS test pool osd recovery"
+          execution_time: ""
+          suite: "suites/quincy/rados/tier-4_rados_test-pool-osd-recovery.yaml"
+          global-conf: "conf/quincy/rados/11-node-cluster.yaml"
```
Comment (pdhiran) [inline pipeline/metadata/6.0.yaml:817]: The suite needs to be changed to : 13-node-cluster.yaml , as this requires node replacements.

File: pipeline/metadata/6.1.yaml [hunk 3]
```diff
@@ -756,6 +756,36 @@ pipelines:
...
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - rados
+        - name: "Tier-4 RADOS test pool osd recovery"
+          execution_time: ""
+          suite: "suites/quincy/rados/tier-4_rados_test-pool-osd-recovery.yaml"
+          global-conf: "conf/quincy/rados/11-node-cluster.yaml"
```
Comment (pdhiran) [inline pipeline/metadata/6.1.yaml:782]: Same as above.

**Review by pdhiran — COMMENTED** [review]
Looks good with minor suite changes needed.

File: pipeline/metadata/5.3.yaml (hunk 1 above)
Comment (neha-gangadhar) [reply pipeline/metadata/5.3.yaml]: sure, done.

File: pipeline/metadata/6.0.yaml (hunk 2 above)
Comment (neha-gangadhar) [reply pipeline/metadata/6.0.yaml]: sure, done.

File: pipeline/metadata/6.1.yaml (hunk 3 above)
Comment (neha-gangadhar) [reply pipeline/metadata/6.1.yaml]: sure, done.

**Review by pdhiran — DISMISSED** [review]
Looks good!

**Review by pdhiran — APPROVED** [review]
Looks Good!

**Pull request merged**